www-apps/jellyfin-bin/jellyfin-bin-10.11.0_rc1-r1.ebuild

```diff
@@ -79,4 +79,12 @@ src_install() {
 
 pkg_postinst() {
 	tmpfiles_process jellyfin.conf
+
+	ewarn "If you are upgrading from a previous version jellyfin will automatically"
```

Comment:
I'll merge this to get it in, but we may want to use `ver_replacing` here.

Reply:
Thanks, I didn't know about that, I'll look into it. I will probably bump the ebuild to rc2 as well as it was released shortly after I created this PR yesterday.